### DISCUSSION TITLE
How do I run this?

### DISCUSSION THREAD

**Discussion by YDJ00** [official-stockfish/Stockfish#4885] [Q&A] [answered]
I have no idea about how to run this, I am new at coding. I have a college project to do on this. Can somebody explain it?

**Comment by dav1312** [comment, marked as answer]
You should take a look at the Wiki: https://github.com/official-stockfish/Stockfish/wiki

Here are some specific pages that might interest you:
- https://github.com/official-stockfish/Stockfish/wiki/Stockfish-FAQ#how-can-i-use-stockfish-in-my-own-software-application
- https://github.com/official-stockfish/Stockfish/wiki/UCI-&-Commands
- https://github.com/official-stockfish/Stockfish/wiki/Download-and-usage
